2-ui/1-document/07-modifying-document/article.md

```diff
@@ -402,40 +402,40 @@ ul.append(...getListContent()); // append + "..." operator = friends!
 </script>
 ```
 
-We mention `DocumentFragment` mainly because there are some concepts on top of it, like [template](info:template-element) element, that we'll cover much later.
+Temel olarak `DocumentFragment` ifadesinden bahsediyoruz. çünkü bunun üzerinde [template](info:template-element) element, gibi bazı kavramlar vardır, ki daha sonra bunlari ele alacağız. 
 
 
-## Removal methods
+## Removal methodsğ
 
-To remove nodes, there are the following methods:
+Düğümleri kaldırmak için, aşaıdaki yöntemler vardır:
```

Comment:
aşağıdaki*

Reply:
Yanlışlarımı düzelttiğiniz için teşekkür ederim.